src/lib.rs

```diff
@@ -197,10 +221,22 @@ impl View for FlexiLoggerView {
         // The longest line sets the width
         let w = logs
             .iter()
-            .map(|msg| msg.width())
+            .map(|msg|
+                msg.spans().map(|x|
+                    // if the log message contains more than one line,
+                    // only the longest line should be considered
+                    // (definitely not the total content.len())
+                    x.content.split('\n').map(|x| x.width()).max().unwrap()
```

Comment:
Nit: if a `span` is empty, this would panic. "Spans are never empty" is a possible invariant, but a bit less obvious to validate.
Maybe `unwrap_or(0)` instead?

Reply:
In that case `"".split('\n')` would consist of `""` and `.max()` would return Some(0).